routes/index.js

```diff
@@ -1,8 +1,9 @@
 const express = require('express')
 const categoryRoutes = require('./categoryRoutes')
+const users = require("./user.routes")
 const router = express.Router()
 
-router.use('/', categoryRoutes)
+router.use('/', categoryRoutes, users)
```

Comment:
remove categorysRoutes, only put users like controller

Reply:
the changes are already resolved